<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>278</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Пул май 2024</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Подсказка 1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="hint1_le"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Подсказка 2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="hint2_le"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Подсказка 3</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="hint3_le"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="answer_pte"/>
    </item>
    <item>
     <widget class="QPushButton" name="copy_answer_btn">
      <property name="text">
       <string>Копировать ответ</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
